vimeo-networking/src/main/java/com/vimeo/networking/AccountStore.java

```diff
@@ -24,25 +24,37 @@
 
 import com.vimeo.networking.model.VimeoAccount;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Interface responsible for handling the creation, deletion, and loading of Vimeo accounts on the client.
  * <p/>
  * Created by hanssena on 4/27/15.
  */
 public interface AccountStore {
 
+    /**
+     * Load the previously saved {@link VimeoAccount} and return it
+     *
+     * @return null if no VimeoAccount could be loaded
+     */
+    @Nullable
     VimeoAccount loadAccount();
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
+     * Saves a {@link VimeoAccount} for loading later.
+     *
+     * @param vimeoAccount the VimeoAccount to save, should not be null when calling this method
+     * @param accountName  the <i>name</i> of the account that helps uniquely identify it in account
+     *                     stores, such as the Android AccountManager
      */
-    @Deprecated
-    void saveAccount(VimeoAccount vimeoAccount, String email, String password);
+    void saveAccount(@NotNull VimeoAccount vimeoAccount, String accountName);
```

Comment:
Sorry to keep at this with the annotations, but I am following up on it since we were discussing this in the context of another PR (https://github.vimeows.com/MobileApps/Vimeo-Android/pull/912). We are going to try to move toward explicitly annotating null and nonnull parameters in method signatures. @alfiehanssen asked us to do it in that context. We've got it on the agenda to discuss for a future Android sync, but in the mean time it makes sense to apply the practice to this PR. It'd be great if you could quickly sweep the files that you touched for this PR and add the annotations. Since there will already be breaking changes in this version. It would be good to review the networking interfaces in general w/r/t annotations and make sure it's all explicit and correct. If it is too much to take on here, we should ticket it, but be sure it gets done before 2.0.0 goes out.